src/components/Card/Card.stories.tsx

```diff
@@ -35,13 +35,6 @@ const CardExample = () => {
   const image = {
     alt: text('alt', 'Image alt text', imageGroupID),
     src: text('src', 'https://placehold.it/600x342', imageGroupID)
-
-    // TODO: handle responsive image sources
-    // sources: {
-    //   thumbnail_lo: text('thumbnail_lo', 'https://placehold.it/300x171', imageGroupID),
-    //   thumbnail: text('thumbnail', 'https://placehold.it/600x342', imageGroupID),
-    //   thumbnail_hi: text('thumbnail_hi', 'https://placehold.it/900x513', imageGroupID)
-    // }
```

Comment:
Have added `imageSrcSet` prop to Card. I'm currently investigating how we can pass these via corporate-react, will create a ticket once I know if it's possible/how to do it.